src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -464,6 +471,110 @@ PRIMARY KEY ({primaryKeys})
             }
         }
 
+        async Task<ScaleMetrics> IScaleMonitor.GetMetricsAsync()
+        {
+            return await this.GetMetricsAsync();
```

Comment:
Why not just put the code directly in this method?

Reply:
The listener needs to expose four methods (two for backward compatibility):

```cs
Task<ScaleMetrics> GetMetricsAsync()
Task<SqlTriggerMetrics> GetMetricsAsync()
ScaleStatus GetScaleStatus(ScaleStatusContext context)
ScaleStatus GetScaleStatus(ScaleStatusContext<SqlTriggerMetrics> context)
```
So, both of the methods are required.